src/components/App/App.css

```diff
@@ -20,4 +20,28 @@
 
 a:link {
   text-decoration: none;
+}
+
+.spinner {
+  margin: auto;
+  /* margin-left: 6em;
+  margin-right: 6em; */
+  display: block;
+  /* float: left; */
+  border: 25px solid rgb(0, 0, 0);
+  width: 1px;
+  height: 1px;
+  border-radius: 50px;
+  border-left-color: transparent;
+  border-right-color: transparent;
+  animation-name: spin;
+  animation-duration: 1.5s;
+  animation-iteration-count: infinite;
+  animation-timing-function: ease-in-out;
```

Comment:
nit: i think there's a shortcut to combine the animation

Reply:
You're probably right! I'll look into this.